src/_tutorials/server/get-started.md

```diff
@@ -1,263 +1,237 @@
 ---
-title: Get started with server-side Dart
-description: Get Dart and run a Dart app.
+title: "Get started: command-line & server apps"
+description: Get Dart, run and compile a small app
+toc: true
 nextpage:
   url: /tutorials/server/cmdline
   title: Write command-line apps
 prevpage:
   url: /tutorials/server
-  title: Server-side Dart tutorials
+  title: Dart command-line and server tutorials
 ---
 
-<div class="mini-toc" markdown="1">
-  <h4>What's the point?</h4>
-
-  * The Dart SDK has development tools and libraries.
-  * Use an IDE or a code editor to create your app.
-  * All Dart apps have a `main()` function.
-  * Dart supports top-level functions.
+Follow these steps to start using Dart to develop command-line & server apps.
+First you’ll play with Dart in your browser, no download required. Then you’ll
+install Dart and run a small program with the Dart VM. Finally, you'll
+AOT-compile that program to native machine code.
+
+## 1. Play with a command-line app in DartPad
```

Comment:
a command-line app -> Dart code

(We don't want to imply that DartPad uses the VM or supports dart:io.)